up-and-runnig-with-recoil-a-new-state-management-library-for-react/03-use-selectors-to-calculate-derived-data-based-on-state-stored-within-a-recoil-atom.md

```diff
@@ -0,0 +1,30 @@
+# Notes
+
+<TimeStamp start="0:27" end="0:29">
```

Comment:
this is also short in duration

"how long does this take to read and process" is the general guideline here.

Reply:
Would it be possible to set a minimum duration for notes displaying with a reasonable default?